docs/v3.x/getting-started/react.md

```diff
@@ -0,0 +1,186 @@
+# Getting Started with React
+
+This integration guide is the following of the [Getting started](../getting-started/quick-start.html). We assume that you have completed the [Step 8](../getting-started/quick-start.html#_8-consume-the-content-type-s-api) and therefore can consume the API by browsing this [url](http://localhost:1337/restaurants).
+
+If you don't come from the Getting started, the way you request a Strapi API with [React](https://reactjs.org/) remains the same except that you will not fetch the same content.
+
+
+### Create a React app
+
+Create a basic React application using [create-react-app](https://reactjs.org/docs/create-a-new-react-app.html).
+
+
+:::: tabs
+
+::: tab yarn
+
+```bash
+yarn create react-app react-app
+```
+
+:::
+
+::: tab npx
+
+```bash
+npx create-react-app react-app
+```
+
+:::
+
+::::
+
+### Use an HTTP client
+
+Many HTTP clients are available but in this documentation we'll use [Axios](https://github.com/axios/axios) and [Fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API)
+
+
+:::: tabs
+
+::: tab axios
+
+```bash
+yarn add axios
+```
+
+:::
+
+::: tab fetch
+
+No installation needed
+
+:::
+
+::::
+
+### GET Request your collection type
+
+Execute a GET request on the `restaurant` Collection Type. 
+
+:::: tabs
+
+::: tab axios
+
+```js
```

Comment:
@lauriejim do we want to include a `require` in this snippet? Or just mention it?

Reply:
You are right! 
`import axios from 'axios';`